scripts/generate.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
It looks like we would still need to manually copy the libs file over. Is that the case?

Reply:
By manually copying the libs file, did you mean having duplicated `lib_*.rego` in `src/<policies-that-import-the-libs>?